### PR TITLE
1.1/441 account page improvements

### PR DESCRIPTION
**Pull request by gingerbreadpup**
A lot of css here to rework the accounts page. Also added a different icon, radio buttons, a new button at the bottom which is only enabled if a radio button is selected, informative copy at the top of the page. Hopefully these will make the page more usable - let me know if you have any other suggestions though.

https://trello.com/c/O7gGmrJ8/441-as-a-trader-i-need-to-understand-my-action-options-on-the-my-account-page

### CODE REVIEW

File: src/pages/Account.vue
Comment (ob6160) [inline src/pages/Account.vue]: This check will have to be changed a little because voucherPayments.length always returns 1. It's because Vue is attaching an Observer I believe.

Comment (ob6160) [reply src/pages/Account.vue]: Probably something like this but in a computed value or even just inline:
`(voucherPayments[0]) ? voucherPayments[0].length > 0 : false`

**Review by ob6160 — DISMISSED** [review]

File: src/pages/Account.vue
Comment (katjam) [reply src/pages/Account.vue]: @gingerbreadpup I thought we already had the function defined - but ah ha to that's for vouchers length

File: src/pages/Account.vue
Comment (katjam) [inline src/pages/Account.vue]: Was it agreed with Kat to have the 2 buttons instead of just giving them one or other option? I think it makes the interface more confusing.

File: src/pages/Account.vue
Comment (gingerbreadpup) [reply src/pages/Account.vue]: Are you sure? I've tested it and it seemed to function correctly.

File: tests/pages/account.test.js
Comment (katjam) [inline tests/pages/account.test.js]: There's an extra ; here

**Review by katjam — COMMENTED** [review]
Couple of bugfixes to go

File: src/pages/Account.vue
Comment (gingerbreadpup) [reply src/pages/Account.vue]: Kat was on board with the two buttons. The button on the right for selected history is greyed out and disabled until a radio button is selected though, which I think helps.

File: src/pages/Account.vue
Comment (katjam) [reply src/pages/Account.vue]: @ob6160 Maybe vue attaches the observer if you observe it but not in normal circumstances?

File: src/pages/Account.vue
Comment (katjam) [reply src/pages/Account.vue]: Yeah - that would help.

File: src/pages/Account.vue
Comment (ob6160) [reply src/pages/Account.vue]: Just tested with Steph and Alex - It's definitely returning an array with the array we want to check the length of inside of it.

Comment (ob6160) [reply src/pages/Account.vue]: image

**Comment by gingerbreadpup** [comment]
Ok. I think this is ready to go. I think @ob6160 is going to do the fix for the voucherPayments.length > 0 bit - not sure how you were planning to do it (in another PR or in here), but I'll leave it up to you.

**Comment by gingerbreadpup** [comment]
Splicing fix works - will merge in once someone approves. Thanks!

**Review by katjam — APPROVED** [review]
Let's get this in!

**Pull request merged**
